tests/functional/settings_common.js

```diff
@@ -26,11 +26,11 @@ define([
 
   var SETTINGS_PAGES = {
     '': 'fxa-settings-header',
-    '/avatar': 'fxa-account-picture-header',
-    '/avatar/change': 'fxa-avatar-change-header',
-    '/avatar/gravatar': 'fxa-avatar-gravatar-header',
-    '/avatar/camera': 'fxa-avatar-camera-header',
-    '/avatar/crop': 'fxa-avatar-crop-header'
+    '/avatar/change': 'avatar-change',
+    '/avatar/gravatar': 'avatar-gravatar',
+    '/avatar/gravatar_permissions': 'avatar-gravatar-permissions',
+    '/avatar/camera': 'avatar-camera',
+    '/avatar/crop': 'avatar-crop'
```

Comment:
It looks like we need to add:

* `communication_preferences`
* `change_password`
* `delete_account`
* `display_name`